applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.cpp

```diff
@@ -40,12 +40,14 @@ void InterfacePreprocessCondition::GenerateInterfacePart<2>(
     
     Parameters default_parameters = Parameters(R"(
     {
-        "simplify_geometry"                    : false
+        "simplify_geometry"                    : false,
+        "contact_property_id"                  : 0
```

Comment:
do you use this for visualization?

Reply:
Nope and yes, indirectly yes, but additionally the contact conditions have their own properties, like friction coefficient

Reply:
Before the properties were created automatically, I just added the possibility of manually set